db/init_db.js

```diff
@@ -1,31 +1,178 @@
 // code to build and initialize DB goes here
 const {
-  client
+  client,
+  createUser,
+  createProducts
   // other db methods 
 } = require('./index');
 
-async function buildTables() {
+const dropTables = async () => {
   try {
-    client.connect();
+    console.log("Starting to drop tables...");
+
+    await client.query(`
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS users;
+      DROP TABLE IF EXISTS products;
+    `);
+    
+
+    console.log("Finished dropping tables!");
+  } catch (error) {
+    console.error("Error dropping tables!");
+    throw error;
+  }
+}
 
-    // drop tables in correct order
+const createTables = async () => {
+  try {
+    console.log("Starting to build tables...");
 
-    // build tables in correct order
+    await client.query(`
+      CREATE TABLE products (
+        id SERIAL PRIMARY KEY,
+        title varchar(255) NOT NULL,
+        author varchar(255) NOT NULL,
+        genre varchar(255) NOT NULL,
+        description varchar(255) NOT NULL,
+        price varchar(255) NOT NULL,
+        "inStock" BOOLEAN NOT NULL DEFAULT false,
+        "imgURL" varchar(255) DEFAULT 'https://www.eduprizeschools.net/wp-content/uploads/2016/06/No_Image_Available.jpg'
+        );
 
+      CREATE TABLE users (
+        id SERIAL PRIMARY KEY,
+        "firstName" varchar(255) NOT NULL,
+        "lastName" varchar(255) NOT NULL,
+        email varchar(255) UNIQUE NOT NULL,
+        "imgURL" varchar(255) DEFAULT 'https://www.eduprizeschools.net/wp-content/uploads/2016/06/No_Image_Available.jpg',
+        username varchar(255) UNIQUE NOT NULL,
+        password varchar(255) NOT NULL,
+        "isAdmin" BOOLEAN DEFAULT false
+        
+      );
+
+      CREATE TABLE orders (
+        id SERIAL PRIMARY KEY,
+        "userId" INTEGER REFERENCES users(id),
+        status varchar(255) DEFAULT 'created',
+        "datePlaced" DATE DEFAULT CURRENT_DATE
+      )
+      `);
+
+
+    console.log("Finished building tables!");
   } catch (error) {
+    console.error("Error building tables!");
     throw error;
   }
 }
 
-async function populateInitialData() {
+async function createInitialProducts() {
+  try {
+
+      console.log("Starting to create products...");
+      await createProducts({
```

Comment:
Maybe add these to an array, and `Promise.all(books)`? Not required, more "cool" factor